cypress/integration/QAM/TC01_basics.spec.js

```diff
@@ -0,0 +1,8 @@
+describe('Login', function(){
+    it('Sign in', function(){
+        cy.visit('https://react-redux.realworld.io/#/login');
+        cy.get(':nth-child(1) > .form-control').type('teuszluczak@gmail.com').should('have.value', 'teuszluczak@gmail.com');
```

Comment:
Dane, które wpisujesz do inputów warto wyrzucić do zmiennych na początku `describe`, szczególnie, jeśli używasz ich kilkukrotnie, tak jak tutaj. Jeśli musisz zmienić, to zmieniasz raz i tyle, nie ma ryzyka, że gdzieś przeoczysz.

Czyli na początku `describe` dajesz np. `const email = 'teuszluczak@gmail.com'`, a potem w `type` i jako drugi argument w `should` podajesz odwołanie do zmiennej, czyli jej nazwę (`email`).

To albo wywalić do JSONa w `fixtures`, wszystko jedno.

Reply:
kumam, spróbuję wdrożyć